client/ayon_photoshop/plugins/publish/collect_review.py

```diff
@@ -24,3 +24,11 @@ def process(self, context):
             if (creator_attributes.get("mark_for_review") and
                     "review" not in instance.data["families"]):
                 instance.data["families"].append("review")
+
+            if "tags" not in instance.data:
+                instance.data["tags"] = []
+
+            if "review" in instance.data["families"]:
+                instance.data["frameStart"] = context.data["frameStart"]
+                instance.data["frameEnd"] = context.data["frameEnd"]
```

Comment:
Shouldn't the frameEnd match the length of output sequence?

Reply:
It does for `mov`, it is updated in `extract_sources_review`.